src/api/config/routes.rb

```diff
@@ -287,6 +287,10 @@ def self.public_or_about_path?(request)
       post 'project/remove_person/:project' => :remove_person, constraints: cons
       post 'project/remove_group/:project' => :remove_group, constraints: cons
       get 'project/monitor/:project' => :monitor, constraints: cons, as: 'project_monitor'
+      # compat route
```

Comment:
I find the comment `compat route` not so great. I would use something like `Backward compatible route`, but I don't want to block the PR for this.

Reply:
I would add a `TODO` to remove this route in the next big OBS release

Reply:
@dmarcoux:

> I find the comment `compat route` not so great. I would use something like `Backward compatible route`, but I don't want to block the PR for this.

I added a commit to change this comment, and all others in the file, to `For backward compatibility`.